src/ui/server/helpers/summary-lists/generate-field-group-item/index.ts

```diff
@@ -7,17 +7,17 @@ import { SummaryListItemData, SummaryListItemDataInput } from '../../../../types
  * getSummaryListItemDataValue
  * Get a field's value from a list of data
  * Conditonally returns a custom value if passed (e.g, custom HTML for an address).
- * @param {Object} Submitted data
  * @param {String} Field ID
+ * @param {Object} Submitted data
  * @param {String} Custom field value
- * @returns {String} Field value
+ * @returns {String} Field value or default empty string dash
  */
-export const getSummaryListItemDataValue = (data: object, fieldId: string, customValue?: string): string => {
+export const getSummaryListItemDataValue = (fieldId: string, data?: object, customValue?: string): string => {
   if (customValue) {
     return customValue;
   }
 
-  if (objectHasProperty(data, fieldId)) {
+  if (data && objectHasProperty(data, fieldId)) {
```

Comment:
we could add this condition to check if object exists in `objectHasProperty`?

Reply:
I don't think we should - the object property in `objectHasProperty` is a required parameter, so it won't compile if an object is _not_ provided, don't think we should change this.

Whereas in this function (`getSummaryListItemDataValue`), the data object is an optional parameter so we have the check in here